packages/editor-ui/src/composables/useCanvasOperations.ts

```diff
@@ -395,6 +395,10 @@ export function useCanvasOperations({ router }: { router: ReturnType<typeof useR
 		ndvStore.activeNodeName = name;
 	}
 
+	function clearNodeActive() {
+		ndvStore.activeNodeName = '';
```

Comment:
Should we use `null` instead of `''` here? I don't think this causes any problems but all other code sets it to `null` to clear

Reply:
Yes, good catch!